recipes/sweep/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "SWEEP" %}
+{% set version = "20160812" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  url: https://github.com/jclev-uga/SWEEP/archive/4282cf9cbf6f6c14993a6ce59d6ed889cd683169.tar.gz
+  md5: ac2c3f721bc852dc5cc622a1b47d20b7 
+
+build:
+  number: 0
+  skip: True # [not py27]
+  skip: True # [osx]
+
+requirements:
+  build:
+    - perl
+    - perl-app-cpanminus
+    - perl-module-build
+
+  run:
+    - python
+    - pysam
+    - biopython
+    - perl
+    - perl-app-cpanminus
+    - perl-module-build
```

Comment:
`perl-app-cpanminus` and `perl-module-build` are not needed as run requirements.